### ISSUE TITLE
Submitting Data to Active Campaign Dropdown does not work properly

### ISSUE THREAD

**Issue by KasimMahroof** [verbb/formie#1540]
### Describe the bug

If you have a dropdown field in Active Campaign, you submit the form with your field mapped to that dropdown field, it will appear as 'Array' instead one of the dropdown values:

![Screenshot 2023-08-21 at 13 27 56](https://github.com/verbb/formie/assets/29715207/6a5b34ec-e5ff-472e-b343-b96ed9baa6ec)

### Steps to reproduce

1. Create form with Active Campaign integration activated (Email Marketing one)
2. In the Active Campaign integration, select list to create contacts on
3. Map data to a dropdown field in that list
4. Submit form data 

### Form settings

- Multi-page form: Yes
- Submission Method: Ajax
- Client-side Validation: Yes 
- Custom Form Templates: No

### Craft CMS version

4.4.15

### Plugin version

2.0.33

### Multi-site?

No

### Additional context

Oddly enough, when I click into the Submission afterwards and then select 'Active Campaign' in the integration dropdown, it then works:
![Screenshot 2023-08-21 at 13 59 15](https://github.com/verbb/formie/assets/29715207/82d3cbcb-351b-4bfc-9bf7-9cd7b75b3f14)

Not sure if Formie is perhaps not formatting the request data correctly the first time or if it's a bug with Active Campaign's API.

**Comment by engram-design** [comment]
Just want to confirm are you're mapping a Formie Dropdown field to an ActiveCampaign Dropdown field? The only reason it should be mapped as an array-like value would be if it's configured to be a multi-value Dropdown field.

Just double-checked on my end, and things seem to be working correctly for me.

<img width="1043" alt="image" src="https://github.com/verbb/formie/assets/1221575/bb33ec41-7543-49db-bc86-dc804a71da91">


And in ActiveCampaign 

<img width="388" alt="image" src="https://github.com/verbb/formie/assets/1221575/ec45f6e5-00a1-4976-b9d9-5b7fd3e7c9c5">

**Comment by KasimMahroof** [comment]
Hi Josh,

Yep, we are mapping a Formie Dropdown field to an ActiveCampaign Dropdown field.

I even tried updating Formie (to 2.0.34.1) and Craft (to 4.4.17) and it was still happening. 

I also created a new test Active Campaign account just to sanity check the problem and the issue still persists.

See what I did below:

Formie dropdown:

![Screenshot 2023-08-22 at 11 35 02](https://github.com/verbb/formie/assets/29715207/543db892-2021-4e18-a7b5-2b7e26c2d84c)

Active Campaign Dropdown:

![Screenshot 2023-08-22 at 11 32 16](https://github.com/verbb/formie/assets/29715207/0f21e778-4d6b-421e-8043-6eba3665296c)

Integration Tab:

![Screenshot 2023-08-22 at 11 35 43](https://github.com/verbb/formie/assets/29715207/435d7b8c-2bae-4d5a-9373-88fe23a50a28)

It's still appearing as 'Array' when we submit the form:

![Screenshot 2023-08-22 at 11 37 42](https://github.com/verbb/formie/assets/29715207/57b1abe3-5290-48a3-996e-2cb093b231ff)

And we then go to the submission, select 'Active Campaign' in the integrations dropdown on the right, it then suddenly works:

![Screenshot 2023-08-22 at 11 39 09](https://github.com/verbb/formie/assets/29715207/9aaadd6d-502b-4113-855d-a4a871af6c20)

![Screenshot 2023-08-22 at 11 39 26](https://github.com/verbb/formie/assets/29715207/5f24b483-a72b-44fe-a0cf-46ed49cf4a1d)

**Comment by KasimMahroof** [comment]
Hi Josh,

After some further digging, it seems like when you submit a Formie Dropdown field it submits it in a an array format.

I modified the formie composer package locally and did some logging of the payload and it would seem that it's formatted like this:

```
"fieldValues":[{"field":4,"value":["Discuss a project"]
```

I then hardcoded the value of the payload value to a string before it got sent and that seemed to have fixed my issue (emailmarketing/ActiveCampaign.php:132):

```
$payload['contact']['fieldValues'][0]['value'] = 'Discuss a project';
```

![Screenshot 2023-08-22 at 17 04 53](https://github.com/verbb/formie/assets/29715207/d56bf5fa-18e0-40cf-b889-bbc3150d3060)

Is this something you can look at fixing?

**Comment by engram-design** [comment]
Ah, I see the issue now, I'm not _quite_ sure why I'm casting the Dropdown field as an [array](https://github.com/verbb/formie/blob/088e19b55133f4eff22b5c3d2310fdc7080d5931/src/integrations/emailmarketing/ActiveCampaign.php#L252), but it shouldn't be.

Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.34.1"`. 

Be sure to also refresh the integration by clicking the refresh button next to the list. Otherwise it'll still treat the dropdown as an array, due to it's caching mechanism.

**Comment by KasimMahroof** [comment]
Hi Josh,

Yup, that fixed it!

Much appreciated for getting a fix in so soon.

**Comment by engram-design** [comment]
Fixed in [2.0.35](https://github.com/verbb/formie/releases/tag/2.0.35)
